### PR TITLE
Highlight related node when a link is hovered over

### PR DESCRIPTION
**Pull request by slakh96**
- This ticket completes https://github.com/orgs/VerWiki/projects/1#card-55326370
- When a link is hovered over in the infoViewer, all elements of the tree including the links, labels, and nodes become greyed out unless they have the same link as the link that is being hovered over

### CODE REVIEW

File: frontend/src/components/Tree/Tree.jsx [hunk 1]
```diff
@@ -100,7 +109,16 @@ function renderTree(dimensions, jsonData, svgRef, onNodeClick, onRightClick) {
...
     )
     .attr("fill", (d) => (d.data.numChildren === 0 ? "#b30000" : "#555"))
-    .attr("r", 6);
+    .attr("r", 6)
+    .attr("opacity", (d) => {
+      if (hoveredNodeLink === "") {
+        return 1;
+      } else if (d.data.url === hoveredNodeLink) {
+        return 1;
+      } else {
```
Comment (rafiiary) [inline frontend/src/components/Tree/Tree.jsx]: You can remove the else clause here and just return 0.25

File: frontend/src/components/Tree/Tree.jsx [hunk 2]
```diff
@@ -125,6 +143,15 @@ function renderTree(dimensions, jsonData, svgRef, onNodeClick, onRightClick) {
     .attr("text-anchor", (d) =>
       d.x < Math.PI === !d.children ? "start" : "end"
     )
+    .attr("opacity", (d) => {
+      if (hoveredNodeLink === "") {
+        return 1;
+      } else if (d.data.url === hoveredNodeLink) {
+        return 1;
+      } else {
```
Comment (rafiiary) [inline frontend/src/components/Tree/Tree.jsx]: Same here please

**Review by rafiiary — CHANGES_REQUESTED** [review]

File: frontend/src/components/Tree/Tree.jsx (hunk 1 above)
Comment (slakh96) [reply frontend/src/components/Tree/Tree.jsx]: fixed

File: frontend/src/components/Tree/Tree.jsx (hunk 2 above)
Comment (slakh96) [reply frontend/src/components/Tree/Tree.jsx]: done

**Review by rafiiary — APPROVED** [review]

**Pull request merged**